apps/designer/app/designer/shared/image-manager/image.tsx

```diff
@@ -0,0 +1,73 @@
+import { useMemo } from "react";
+import { styled } from "@webstudio-is/design-system";
+import { Image as WebstudioImage, loaders } from "@webstudio-is/image";
+import env from "~/shared/env";
+import type { RenderableAsset } from "../assets";
+import brokenImage from "~/shared/images/broken-image-placeholder.svg";
+
+type ImageProps = {
+  renderableAsset: RenderableAsset;
+  alt: string;
+  width: number;
+};
+
+const StyledWebstudioImage = styled(WebstudioImage, {
+  position: "absolute",
+  width: "100%",
+  height: "100%",
+  objectFit: "contain",
+
+  // This is shown only if image was not loaded and broken
+  // from spec:
+  // The pseudo-elements generated by ::before and ::after are contained by the element's formatting box,
+  // and thus don't apply to replaced elements such as <img>, or to <br> elements
+  // But broken image is replaced element so this style is applied
```

Comment:
Should the comment say "But broken image is **not** a replaced element"? Otherwise there's a contradiction with the previous text